test/built-ins/String/prototype/trimEnd/this-value-boolean.js

```diff
@@ -0,0 +1,21 @@
+// Copyright (c) 2017 Valerie Young.  All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+esid: pending
+description: Behavoir when "this" value is a boolean.
```

Comment:
typo on Behavior.

missing info and feature tags.

Reply:
I thought "info" wasn't mandatory? Especially because this is a success case, I wasn't sure what text from ecmascript to include..

Reply:
while it's not mandatory, we are trying to enforce its use with the matching spec text for the unit being tested. It helps a ton while reading the test with a cross reference.

Reply:
``` 
Runtime Semantics: TrimString ( string, where )

1. Let str be ? RequireObjectCoercible(string).
2. Let S be ? ToString(str). 
...

ToString ( argument )

- If argument is true, return "true".
- If argument is false, return "false".
```

Reply:
Later Friday day, after making this comment, I came to the same conclusion as you! Commit here: 
https://github.com/tc39/test262/pull/1246/commits/4587c8136929c078e85747ecfdc7318a7d5dc2a1

But I didn't include quite as much information, "TrimString" instead of "Runtime Semantics: TrimString (string, where)". I think I should add the extra information for extra clarity.